src/porepy/models/constitutive_laws.py

```diff
@@ -1627,17 +1349,19 @@ def diffusive_flux(
             # Note: in case of an empty subdomain list, the time dependent array is
             # still returned. Otherwise, this method produces an infinite recursion
             # loop. It does not affect real computations.
-            return self.create_boundary_operator(  # type: ignore[call-arg]
+            domains = cast(Sequence[pp.BoundaryGrid], domains)
+            return self.create_boundary_operator(
                 name=flux_name,
                 domains=domains,
             )
+        domains = list(cast(Sequence[pp.Grid], domains))
```

Comment:
For my own education, any reason to do the conversion list->Sequence->list?

Reply:
Removed it